cmd/tikv-ctl/src/executor.rs

```diff
@@ -1135,6 +1186,19 @@ impl<ER: RaftEngine> DebugExecutor for DebuggerImpl<ER> {
     fn reset_to_version(&self, version: u64) {
         Debugger::reset_to_version(self, version);
     }
+
+    fn flashback_to_version(
+        &self,
+        _version: u64,
+        _region_id: u64,
+
+        _start_key: Vec<u8>,
+        _end_key: Vec<u8>,
+        _start_ts: u64,
+        _commit_ts: u64,
+    ) -> Result<FlashbackToVersionResponse> {
+        unimplemented!("only available for online mode");
```

Comment:
let's call it remote mode

Reply:
just keep consistent with others, should I change all of them?

Reply:
some others call it remote mode as well, please change all of them

Reply:
done.